fe/fe-core/src/main/java/org/apache/doris/planner/StreamLoadPlanner.java

```diff
@@ -138,20 +141,22 @@ public TExecPlanFragmentParams plan(TUniqueId loadId) throws UserException {
             slotDesc.setColumn(col);
             slotDesc.setIsNullable(col.isAllowNull());
 
-            SlotDescriptor scanSlotDesc = descTable.addSlotDescriptor(scanTupleDesc);
-            scanSlotDesc.setIsMaterialized(true);
-            scanSlotDesc.setColumn(col);
-            scanSlotDesc.setIsNullable(col.isAllowNull());
-            for (ImportColumnDesc importColumnDesc : taskInfo.getColumnExprDescs().descs) {
-                try {
-                    if (!importColumnDesc.isColumn() && importColumnDesc.getColumnName() != null
-                            && importColumnDesc.getColumnName().equals(col.getName())) {
-                        scanSlotDesc.setIsNullable(importColumnDesc.getExpr().isNullable());
-                        break;
+            if (Config.enable_vectorized_load) {
```

Comment:
Use `VectorizedUtil.isVectorized()`

Reply:
`VectorizedUtil.isVectorized()` is only for query